src/pages/state/[state].js

```diff
@@ -224,6 +210,20 @@ export default function State({ data: { statesGeoData }, state = "DC" }) {
             state={stateData.abbreviation}
           />
           <EquitySection size="sm" />
+          {stateData.videos?.length ? (
+            <Row className={styles.voices}>
+              <Row>
+                <h3>Voices For Reform</h3>
+              </Row>
+              {stateData.videos.map(({ link, title, description }, i) => (
+                <Row key={`statewide-videos-${i}`}>
+                  <ReactPlayer url={link} width={594} height={361} />
```

Comment:
Can we pass `width="100%"` and find a way to make that work in style? (And `height` should not need to be set)

Reply:
Just realized this is in `main` as is. Still appreciate the change!